release/alpa_tests/app_config.yaml

```
@@ -0,0 +1,34 @@
+base_image: {{ env["RAY_IMAGE_ML_NIGHTLY_GPU"] | default("anyscale/ray-ml:nightly-py39-gpu") }}
```

```suggestion
base_image: {{ env["RAY_IMAGE_ML_NIGHTLY_GPU"] | default("anyscale/ray-ml:nightly-py37-gpu") }}
```

Just FYI, this will be overwritten with the `py37` image per default.

If you want to use Python 3.9, you'll have to specify it in `release_tests.yaml`